src/views/landingPage/Index.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect } from 'react';
+import {
+  Container,
+  CardDeck,
+  CardColumns,
+  Row,
+  Col,
+  Button,
+} from 'react-bootstrap';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Banner from '../../components/banner/Index';
+import { getArticles, getTrendingArticles } from '../../redux/actions/getArticles';
+import ArticleSummary from '../../components/articleSummary/Index';
+import RectArticleSummary from '../../components/rectArticleSummary/Index';
+import ContentHeader from '../../components/contentHeader/Index';
+import './landingPage.scss';
+import Footer from '../../components/footer/Index';
+
+
+const LandingPage = (props) => {
+  const { history, trendingArticles, articles } = props;
+  useEffect(() => {
+    props.getArticles(3).then();
+  }, {});
+
+  useEffect(() => {
+    props.getTrendingArticles(6).then();
+  }, {});
+
+  // const ArticleSummaryStack = () => props.trendingArticles.map(data => (
```

Comment:
@olajide1234 why do we have commented out code here?

Reply:
Removed now.